### PR TITLE
Add gauge to time mark duration

### PR DESCRIPTION
**Pull request by RatanRSur**
This is required to do performance monitoring/improvement of mark duration

### CODE REVIEW

File: ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java [hunk 1]
```diff
@@ -94,6 +97,13 @@ public MarkSweepPruner(
             BesuMetricCategory.PRUNER,
             "sweep_operations_total",
             "Total number of sweep operations performed");
+
+    markStopwatch = Stopwatch.createUnstarted();
+    metricsSystem.createLongGauge(
+        BesuMetricCategory.PRUNER,
+        "mark_time_duration",
+        "Total number of seconds spent marking the state trie",
```
Comment (shemnon) [inline ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java:105]: Cumulative or per marking run?  Worth adding some verbiage to the help text to make it clear.

File: ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java [hunk 2]
```diff
@@ -94,6 +97,13 @@ public MarkSweepPruner(
             BesuMetricCategory.PRUNER,
             "sweep_operations_total",
             "Total number of sweep operations performed");
+
+    markStopwatch = Stopwatch.createUnstarted();
+    metricsSystem.createLongGauge(
```
Comment (shemnon) [inline ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java:102]: Is MarkSweepPruner a singleton for the life of the java process?  Counters can handle multiple counters, gauges will throw an exception when the second gauge is created.

Comment (RatanRSur) [reply ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java:102]: Yup, it's a singleton

File: ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java (hunk 1 above)
Comment (RatanRSur) [reply ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPruner.java]: resolved

**Pull request merged**